src/Our.Umbraco.EnvironmentDashboard/Controllers/EnvironmentDashboardController.cs

```diff
@@ -1,21 +1,40 @@
-﻿using System.Net;
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net;
 using System.Net.Http;
 using System.Web.Http;
+using Our.Umbraco.EnvironmentDashboard.Components;
 using Our.Umbraco.EnvironmentDashboard.Models;
 using Umbraco.Web.WebApi;
 
 namespace Our.Umbraco.EnvironmentDashboard.Controllers
 {
 	public class EnvironmentDashboardController : UmbracoAuthorizedApiController
 	{
+		private readonly IDashboardEnvironmentProvider _environmentProvider;
+		private readonly EnvironmentDashboardGroupsCollection _groupsCollection;
+		
+		public EnvironmentDashboardController(IDashboardEnvironmentProvider environmentProvider, EnvironmentDashboardGroupsCollection groupsCollection)
+		{
+			_groupsCollection = groupsCollection;
+			_environmentProvider = environmentProvider;
+		}
```

Comment:
I think the groups should live as a property of some the dashboard classes. That makes more sense IMO.

Reply:
I think there should be a DashboardService, that returns a Dashboard class that the actual model returned, so that the Controller only executes the logic to return it as an http response